include/obstacle_detector/utilities/segment.h

```diff
@@ -107,6 +107,15 @@ class Segment
     return (p - projection).length();
   }
 
+  bool isBetweenEndpoints(const Point& p, double tol) const {
+    // Checks if point is between segment endpings (within a tolerance to account for noise)
+    Point a = last_point - first_point;
+    Point b = p - first_point;
+    Point c = p - last_point;
```

Comment:
what is c used for?

Reply:
Good catch, I was using it for something else before, and forgot to remove it.

Reply:
please don't use single letter variables